client/src/components/DetailCard.js

```diff
@@ -0,0 +1,51 @@
+import React from "react";
+import styled from "styled-components/macro";
+import restaurant from "../stories/assets/restaurant.png";
+
+const FinalCardContainer = styled.div`
```

Comment:
these components look the same as the decision components. you might wanna add a "general" component instead

Reply:
Thank you! would you explain which component you referring to? I can't figure out which one you mean